docs/scanners.md

```diff
@@ -268,6 +268,9 @@ Scan rules:
 110007  Username Disclosure [Script]
 110008  Suspicious XML Comments Disclosure [Script]
 
+120000  Information Disclosure - Information in Browser Storage
+120001  Information Disclosure - Sensitive Information in Browser Storage
```

Comment:
What's the distinction here? Just reading titles this sounds like a perfect place to use Alert Refs.

Reply:
The first is for _any_ local / session storage event .. and I will be using alertRefs for those. These will be raised at Info level. These are not vulns, they just help you understand the app better
The second is a rip off of InformationDisclosureInUrlScanRule and will be raised at Low.
I think they should be separate so people can turn one or other off - thats much harder if they have the same IDs.